cert.go

```diff
@@ -145,6 +145,10 @@ func loadCAFromConfig(l *logrus.Logger, c *config.C) (*cert.NebulaCAPool, error)
 	}
 
 	CAs, err := cert.NewCAPoolFromBytes(rawCA)
+	if errors.Is(err, cert.ErrExpired) {
+		l.WithError(err).Warn("expired certificates present in CA pool")
```

Comment:
This results in a silly looking log message.

Warning with a single expired CA:
```WARN[0000] expired certificates present in CA pool       error="$name1: certificate is expired: certificate is expired"```

The warning when two CAs are invalid:
```WARN[0000] expired certificates present in CA pool       error="$name1: certificate is expired,$name2: certificate is expired: certificate is expired"```

Reply:
I think what we probably want at the end of the day is to return an error from `cert.NewCAPoolFromBytes` that contains all the expired certificates and then in this check here we iterate them and log individual warnings for each one. That way we can output the full certificate details rather than just the name.

Reply:
Pushed commits